pkg/driver/grpc.go

```diff
@@ -43,16 +44,41 @@ func parseEndpoint(ep string) (string, string, error) {
 	return "", "", fmt.Errorf("Invalid endpoint: %v", ep)
 }
 
+//filters if the logd are informative or pollutant
+func infotrmativeLog(info *grpc.UnaryServerInfo) bool {
+
+	// add the messages that pollute logs to the array
+	var msgsToFilter = [][]byte{
+		[]byte("NodeGetVolumeStats"),
+		[]byte("NodeGetCapabilities"),
+	}
+
+	// checks for message in request
+	for _, msg := range msgsToFilter {
+		if bytes.Contains([]byte(info.FullMethod), msg) {
+			return false
+		}
+	}
+
+	return true
+}
+
 // logGRPC logs all the grpc related errors, i.e the final errors
 // which are returned to the grpc clients
 func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
-	logrus.Infof("GRPC call: %s", info.FullMethod)
-	logrus.Infof("GRPC request: %s", protosanitizer.StripSecrets(req))
+
+	logRequest := infotrmativeLog(info)
+	if logRequest == true {
+		klog.Infof("GRPC call: %s", info.FullMethod)
```

Comment:
let us club both klog statement in one statement with new line.

klog.Infof("GRPC call: %s\n request: %s", info.FullMethod, protosanitizer.StripSecrets(req))